bedrock/firefox/templates/firefox/election/index.html

```diff
@@ -0,0 +1,167 @@
+  {{ css_bundle('firefox_election') }}
+{% endblock %}
+
+{% block body_id %}election{% endblock %}
+
+{% block body_class %}election{% endblock %}
+
+{% block content %}
+
+<main class="mzp-t-firefox">
+  <div id="case-not-firefox-desktop" class="hidden">
+    <div class="t-paper">
+      <section class="mzp-c-hero">
+        <div class="mzp-l-content">
+          <div class="mzp-c-hero-body">
+            <h1 class="mzp-c-hero-title">Get Smart Before You Vote.</h1>
+            <div class="mzp-c-hero-desc">
+              <p>Decode political ad targeting and keep Facebook off your trail with these extensions.</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="mzp-c-call-out mzp-t-product-firefox mzp-t-firefox t-container">
+        <div class="mzp-l-content">
+          <h2 class="mzp-c-call-out-title">The Firefox Facebook Container</h2>
+          <div class="mzp-c-call-out-desc">
+            <p>Facebook lets political advertisers target you by tracking your activity on and off their platform.</p>
+            <p>With this extension, Firefox makes it easier to explore the web without Facebook watching, so you can keep your political opinions to yourself and your dinner companions.</p>
+            <p class="mzp-c-button-download">
+              <small class="mzp-c-button-download-privacy-link">
+                <a href="{{ url('privacy.notices.firefox') }}">Firefox Privacy Notice</a>
+              </small>
+            </p>
+          </div>
+        </div>
+      </section>
+    </div>
+    <section class="mzp-c-call-out mzp-t-product-firefox mzp-t-firefox t-pro">
+      <div class="mzp-l-content">
+        <h2 class="mzp-c-call-out-title">The ProPublica Facebook Political Ad Collector</h2>
+        <div class="mzp-c-call-out-desc">
+          <p>Participate in ProPublica’s crowdsourcing extension that automatically gathers ads from Facebook, the biggest online platform for political discourse. Your contribution feeds a public database that lets you see what others are seeing and helps journalists flag misinformation. It’s a win-win for you and the web.</p>
+          <p>Since the Political Ad Collector was developed by ProPublica, not Firefox, the <a rel="external" href="https://addons.mozilla.org/en-US/firefox/addon/facebook-ad-collector/privacy/">ProPublica privacy policy</a> applies to this extension.</p>
+        </div>
+      </div>
+    </section>
+    <section id="case-unsupported" class="mzp-c-call-out mzp-t-firefox hidden">
+      <div class="mzp-l-content">
+        <div class="mzp-c-call-out-cta">
+          <div class="mzp-c-call-out-cta-container">
+            <div class="mzp-c-button-download">
+              <div>
+                <p>Sad to say, your computer doesn’t support Firefox.  Even so, Digital citizens need good info. Mozilla Corporation and Firefox can help you check your registration status, read political news curated by Pocket and get other helpful resources.</p>
```

Comment:
I'm seeing this message on Android rather than the mobile case below. I'm not sure why...